unexpected EOF when running example template

### Terraform Version

Terraform v0.11.3
+ provider.vsphere v1.3.2

### vSphere Provider Version

$ terraform providers
.
└── provider.vsphere

### Affected Resource(s)

- `vsphere_virtual_machine`

### Terraform Configuration Files

Based on template from https://www.terraform.io/docs/providers/vsphere/index.html

```hcl
provider "vsphere" {
  user           = "<user>"
  password       = "<password>"
  vsphere_server = "<ip>"

  # if you have a self-signed cert
  allow_unverified_ssl = true

  client_debug = true
}

data "vsphere_datacenter" "dc" {
  name = "Cloud-DC"
}

data "vsphere_datastore" "datastore" {
  name          = "datastore1"
  datacenter_id = "${data.vsphere_datacenter.dc.id}"
}

data "vsphere_resource_pool" "pool" {
  name          = "Cluster-1/Resources"
  datacenter_id = "${data.vsphere_datacenter.dc.id}"
}

data "vsphere_network" "network" {
  name          = "VM Network"
  datacenter_id = "${data.vsphere_datacenter.dc.id}"
}

resource "vsphere_virtual_machine" "vm" {
  name             = "terraform-test"
  resource_pool_id = "${data.vsphere_resource_pool.pool.id}"
  datastore_id     = "${data.vsphere_datastore.datastore.id}"

  num_cpus = 2
  memory   = 1024
  guest_id = "other3xLinux64Guest"

  network_interface {
    network_id = "${data.vsphere_network.network.id}"
  }

  disk {
    label = "disk0"
    size  = 20
  }
}
```

### Crash Output

```
Error: Error applying plan:

1 error(s) occurred:

* vsphere_virtual_machine.vm: 1 error(s) occurred:

* vsphere_virtual_machine.vm: unexpected EOF

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.


panic: runtime error: invalid memory address or nil pointer dereference
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xf79f13]
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: goroutine 193 [running]:
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vsphere/inter$
al/helper/computeresource.DefaultDevicesFromReference(0xc42053a070, 0xc4205bc220, 0x16, 0xc4205a5390, 0xa, 0xc420014fe0, 0x13, 0x0, 0x0, 0x0, ...)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terrafor$
-providers/terraform-provider-vsphere/vsphere/internal/helper/computeresource/compute_resource_helper.go:146 +0x1c3
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vsphere/inter$
al/helper/resourcepool.DefaultDevices(0xc42053a070, 0xc420302040, 0xc420014fe0, 0x13, 0xc4202db0f0, 0xc42049f540, 0xc, 0xc42050d4b4, 0xa)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terrafor$
-providers/terraform-provider-vsphere/vsphere/internal/helper/resourcepool/resource_pool_helper.go:116 +0x229
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vsphere.resou$
ceVSphereVirtualMachineCreateBare(0xc4205ba3f0, 0x1206180, 0xc42016cf40, 0x11760e0, 0xc420143c00, 0x0)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terrafor$
-providers/terraform-provider-vsphere/vsphere/resource_vsphere_virtual_machine.go:674 +0x971
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vsphere.resou$
ceVSphereVirtualMachineCreate(0xc4205ba3f0, 0x1206180, 0xc42016cf40, 0x24, 0x20f8160)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/[37/1849]
-providers/terraform-provider-vsphere/vsphere/resource_vsphere_virtual_machine.go:219 +0x4af
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vendor/github.
com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc42014a690, 0xc42007a140, 0xc420143800, 0x1206180, 0xc42016cf40, 0x1, 0xc42058c2a0, 0xc420068de0
)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform
-providers/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:227 +0x364
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vendor/github.
com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc42014ad90, 0xc42007a050, 0xc42007a140, 0xc420143800, 0x7f5c5c67c640, 0x0, 0x18)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform
-providers/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:283 +0xa4
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: github.com/terraform-providers/terraform-provider-vsphere/vendor/github.
com/hashicorp/terraform/plugin.(*ResourceProviderServer).Apply(0xc4201429e0, 0xc420142780, 0xc4202da240, 0x0, 0x0)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform
-providers/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform/plugin/resource_provider.go:527 +0x57
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: reflect.Value.call(0xc4200688a0, 0xc42018c070, 0x13, 0x148d15f, 0x4, 0xc
42027bf20, 0x3, 0x3, 0x403a23, 0xc42053c000, ...)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /usr/local/go/src/reflect/value.go:434 +0x905
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: reflect.Value.Call(0xc4200688a0, 0xc42018c070, 0x13, 0xc42003ef20, 0x3,
0x3, 0xc42003ef20, 0xc42003ef28, 0xc420354080)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /usr/local/go/src/reflect/value.go:302 +0xa4
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: net/rpc.(*service).call(0xc420558140, 0xc42055e050, 0xc42036a1b8, 0xc420
376000, 0xc420143300, 0x11237e0, 0xc420142780, 0x16, 0x1123820, 0xc4202da240, ...)
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /usr/local/go/src/net/rpc/server.go:381 +0x142
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4: created by net/rpc.(*Server).ServeCodec
2018-02-23T16:29:13.043-0800 [DEBUG] plugin.terraform-provider-vsphere_v1.3.2_x4:       /usr/local/go/src/net/rpc/server.go:475 +0x36b
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalWriteState
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalApplyProvisioners
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalIf
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalWriteState
2018-02-23T16:29:13.046-0800 [DEBUG] plugin: plugin process exited: path=/usr/local/google/home/rdc/go/src/k8s.io/kube-deploy/cluster-api-gcp/cloud/terraf
orm/templates/vmware/.terraform/plugins/linux_amd64/terraform-provider-vsphere_v1.3.2_x4
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalWriteDiff
2018/02/23 16:29:13 [TRACE] root: eval: *terraform.EvalApplyPost
2018/02/23 16:29:13 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error(s) occurred:

* vsphere_virtual_machine.vm: unexpected EOF
2018/02/23 16:29:13 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* vsphere_virtual_machine.vm: unexpected EOF
2018/02/23 16:29:13 [TRACE] [walkApply] Exiting eval tree: vsphere_virtual_machine.vm
2018/02/23 16:29:13 [TRACE] dag/walk: upstream errored, not walking "meta.count-boundary (count boundary fixup)"
2018/02/23 16:29:13 [TRACE] dag/walk: upstream errored, not walking "provider.vsphere (close)"
2018/02/23 16:29:13 [TRACE] dag/walk: upstream errored, not walking "root"
2018/02/23 16:29:13 [TRACE] Preserving existing state lineage "e64226d1-582d-4051-acfb-f7cff06660bb"
2018/02/23 16:29:13 [TRACE] Preserving existing state lineage "e64226d1-582d-4051-acfb-f7cff06660bb"
2018/02/23 16:29:13 [TRACE] Preserving existing state lineage "e64226d1-582d-4051-acfb-f7cff06660bb"
2018/02/23 16:29:13 [TRACE] Preserving existing state lineage "e64226d1-582d-4051-acfb-f7cff06660bb"
2018/02/23 16:29:13 [DEBUG] plugin: waiting for all plugin processes to complete...
2018-02-23T16:29:13.056-0800 [WARN ] plugin: error closing client during Kill: err="connection is shut down"



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

### Expected Behavior

Crashed

### Actual Behavior

I would have expected the VM creation or some better error message.

### Steps to Reproduce

1. `terraform apply .`

--- a/tf-vsphere-devrc.mk.example
+++ b/tf-vsphere-devrc.mk.example
@@ -33,6 +33,7 @@ export VSPHERE_IPV4_GATEWAY        ?= 10.0.0.1   # Customization gateway
 export VSPHERE_DNS                 ?= 10.0.0.10  # Customization DNS
 export VSPHERE_DATACENTER          ?= vm-dc      # VM placement DC
 export VSPHERE_CLUSTER             ?= vm-clus1   # VM placement cluster
+export VSPHERE_EMPTY_CLUSTER       ?= vm-clus2   # Empty cluster for testing
 export VSPHERE_RESOURCE_POOL       ?= vm-respool # VM resource resource pool
 export VSPHERE_DATASTORE           ?= datastore1 # VM placement datastore
 export VSPHERE_DATASTORE2          ?= datastore2 # 2nd datastore for vMotion

--- a/vsphere/internal/helper/computeresource/compute_resource_helper.go
+++ b/vsphere/internal/helper/computeresource/compute_resource_helper.go
@@ -32,6 +32,7 @@ type BaseComputeResource interface {
 	Name() string
 	Properties(context.Context, types.ManagedObjectReference, []string, interface{}) error
 	Reference() types.ManagedObjectReference
+	String() string
 }
 
 // StandaloneFromID locates a ComputeResource by its managed object reference ID.
@@ -139,11 +140,10 @@ func BasePropertiesFromReference(client *govmomi.Client, ref types.ManagedObject
 // specific compute resource from a supplied managed object reference.
 func DefaultDevicesFromReference(client *govmomi.Client, ref types.ManagedObjectReference, guest string) (object.VirtualDeviceList, error) {
 	log.Printf("[DEBUG] Fetching default device list for object reference %q for OS type %q", ref.Value, guest)
-	props, err := BasePropertiesFromReference(client, ref)
+	b, err := EnvironmentBrowserFromReference(client, ref)
 	if err != nil {
 		return nil, err
 	}
-	b := envbrowse.NewEnvironmentBrowser(client.Client, *props.EnvironmentBrowser)
 	ctx, cancel := context.WithTimeout(context.Background(), provider.DefaultAPITimeout)
 	defer cancel()
 	return b.DefaultDevices(ctx, "", nil)
@@ -152,12 +152,37 @@ func DefaultDevicesFromReference(client *govmomi.Client, ref types.ManagedObject
 // OSFamily uses the compute resource's environment browser to get the OS family
 // for a specific guest ID.
 func OSFamily(client *govmomi.Client, ref types.ManagedObjectReference, guest string) (string, error) {
-	props, err := BasePropertiesFromReference(client, ref)
+	b, err := EnvironmentBrowserFromReference(client, ref)
 	if err != nil {
 		return "", err
 	}
-	b := envbrowse.NewEnvironmentBrowser(client.Client, *props.EnvironmentBrowser)
 	ctx, cancel := context.WithTimeout(context.Background(), provider.DefaultAPITimeout)
 	defer cancel()
 	return b.OSFamily(ctx, guest)
+}
+
+// EnvironmentBrowserFromReference loads an environment browser for the
+// specific compute resource reference. The reference can be either a
+// standalone host or cluster.
+//
+// As an added safety feature if the compute resource properties come back with
+// an unset environmentBrowser attribute, this function will return an error.
+// This is to protect against cases where this may come up such as licensing
+// issues or clusters without hosts.
+func EnvironmentBrowserFromReference(client *govmomi.Client, ref types.ManagedObjectReference) (*envbrowse.EnvironmentBrowser, error) {
+	cr, err := BaseFromReference(client, ref)
+	if err != nil {
+		return nil, err
+	}
+	props, err := BaseProperties(cr)
+	if err != nil {
+		return nil, err
+	}
+	if props.EnvironmentBrowser == nil {
+		return nil, fmt.Errorf(
+			"compute resource %q is missing an Environment Browser. Check host, cluster, and vSphere license health of all associated resources and try again",
+			cr,
+		)
+	}
+	return envbrowse.NewEnvironmentBrowser(client.Client, *props.EnvironmentBrowser), nil
 }

--- a/vsphere/resource_vsphere_virtual_machine_test.go
+++ b/vsphere/resource_vsphere_virtual_machine_test.go
@@ -791,6 +791,26 @@ func TestAccResourceVSphereVirtualMachine(t *testing.T) {
 			},
 		},
 		{
+			"create into empty cluster, no environment browser",
+			resource.TestCase{
+				PreCheck: func() {
+					testAccPreCheck(tp)
+					testAccResourceVSphereVirtualMachinePreCheck(tp)
+				},
+				Providers: testAccProviders,
+				Steps: []resource.TestStep{
+					{
+						Config:      testAccResourceVSphereVirtualMachineConfigBasicEmptyCluster(),
+						ExpectError: regexp.MustCompile("compute resource .* is missing an Environment Browser\\. Check host, cluster, and vSphere license health of all associated resources and try again"),
+					},
+					{
+						Config: testAccResourceVSphereEmpty,
+						Check:  resource.ComposeTestCheckFunc(),
+					},
+				},
+			},
+		},
+		{
 			"clone from template",
 			resource.TestCase{
 				PreCheck: func() {
@@ -1024,6 +1044,27 @@ func TestAccResourceVSphereVirtualMachine(t *testing.T) {
 					{
 						Config:      testAccResourceVSphereVirtualMachineConfigCloneBadVAppSettings(),
 						ExpectError: regexp.MustCompile("this VM lacks a vApp configuration and cannot have vApp properties set on it"),
+					},
+				},
+			},
+		},
+		{
+			"clone into empty cluster, no environment browser (fails on plan)",
+			resource.TestCase{
+				PreCheck: func() {
+					testAccPreCheck(tp)
+					testAccResourceVSphereVirtualMachinePreCheck(tp)
+				},
+				Providers: testAccProviders,
+				Steps: []resource.TestStep{
+					{
+						Config:      testAccResourceVSphereVirtualMachineConfigCloneEmptyCluster(),
+						ExpectError: regexp.MustCompile("compute resource .* is missing an Environment Browser\\. Check host, cluster, and vSphere license health of all associated resources and try again"),
+						PlanOnly:    true,
+					},
+					{
+						Config: testAccResourceVSphereEmpty,
+						Check:  resource.ComposeTestCheckFunc(),
 					},
 				},
 			},
@@ -8086,5 +8127,190 @@ resource "vsphere_virtual_machine" "vm" {
 		os.Getenv("VSPHERE_DNS"),
 		os.Getenv("VSPHERE_DATASTORE"),
 		os.Getenv("VSPHERE_TEMPLATE"),
+	)
+}
+
+func testAccResourceVSphereVirtualMachineConfigBasicEmptyCluster() string {
+	return fmt.Sprintf(`
+variable "datacenter" {
+  default = "%s"
+}
+
+variable "resource_pool" {
+  default = "%s/Resources"
+}
+
+variable "network_label" {
+  default = "%s"
+}
+
+variable "datastore" {
+  default = "%s"
+}
+
+data "vsphere_datacenter" "dc" {
+  name = "${var.datacenter}"
+}
+
+data "vsphere_datastore" "datastore" {
+  name          = "${var.datastore}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_resource_pool" "pool" {
+  name          = "${var.resource_pool}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_network" "network" {
+  name          = "${var.network_label}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+resource "vsphere_virtual_machine" "vm" {
+  name             = "terraform-test"
+  resource_pool_id = "${data.vsphere_resource_pool.pool.id}"
+  datastore_id     = "${data.vsphere_datastore.datastore.id}"
+
+  num_cpus = 2
+  memory   = 2048
+  guest_id = "other3xLinux64Guest"
+
+  network_interface {
+    network_id = "${data.vsphere_network.network.id}"
+  }
+
+  disk {
+    label = "disk0"
+    size  = 20
+  }
+}
+`,
+		os.Getenv("VSPHERE_DATACENTER"),
+		os.Getenv("VSPHERE_EMPTY_CLUSTER"),
+		os.Getenv("VSPHERE_NETWORK_LABEL_PXE"),
+		os.Getenv("VSPHERE_DATASTORE"),
+	)
+}
+
+func testAccResourceVSphereVirtualMachineConfigCloneEmptyCluster() string {
+	return fmt.Sprintf(`
+variable "datacenter" {
+  default = "%s"
+}
+
+variable "resource_pool" {
+  default = "%s/Resources"
+}
+
+variable "network_label" {
+  default = "%s"
+}
+
+variable "ipv4_address" {
+  default = "%s"
+}
+
+variable "ipv4_netmask" {
+  default = "%s"
+}
+
+variable "ipv4_gateway" {
+  default = "%s"
+}
+
+variable "dns_server" {
+  default = "%s"
+}
+
+variable "datastore" {
+  default = "%s"
+}
+
+variable "template" {
+  default = "%s"
+}
+
+variable "linked_clone" {
+  default = "%s"
+}
+
+data "vsphere_datacenter" "dc" {
+  name = "${var.datacenter}"
+}
+
+data "vsphere_datastore" "datastore" {
+  name          = "${var.datastore}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_resource_pool" "pool" {
+  name          = "${var.resource_pool}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_network" "network" {
+  name          = "${var.network_label}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_virtual_machine" "template" {
+  name          = "${var.template}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+resource "vsphere_virtual_machine" "vm" {
+  name             = "terraform-test"
+  resource_pool_id = "${data.vsphere_resource_pool.pool.id}"
+  datastore_id     = "${data.vsphere_datastore.datastore.id}"
+
+  num_cpus = 2
+  memory   = 2048
+  guest_id = "${data.vsphere_virtual_machine.template.guest_id}"
+
+  network_interface {
+    network_id   = "${data.vsphere_network.network.id}"
+    adapter_type = "${data.vsphere_virtual_machine.template.network_interface_types[0]}"
+  }
+
+  disk {
+    label            = "disk0"
+    size             = "${data.vsphere_virtual_machine.template.disks.0.size}"
+    eagerly_scrub    = "${data.vsphere_virtual_machine.template.disks.0.eagerly_scrub}"
+    thin_provisioned = "${data.vsphere_virtual_machine.template.disks.0.thin_provisioned}"
+  }
+
+  clone {
+    template_uuid = "${data.vsphere_virtual_machine.template.id}"
+    linked_clone  = "${var.linked_clone != "" ? "true" : "false" }"
+
+    customize {
+      linux_options {
+        host_name = "terraform-test"
+        domain    = "test.internal"
+      }
+
+      network_interface {
+        ipv4_address = "${var.ipv4_address}"
+        ipv4_netmask = "${var.ipv4_netmask}"
+      }
+
+      ipv4_gateway    = "${var.ipv4_gateway}"
+      dns_server_list = ["${var.dns_server}"]
+      dns_suffix_list = ["test.internal"]
+    }
+  }
+}
+`,
+		os.Getenv("VSPHERE_DATACENTER"),
+		os.Getenv("VSPHERE_EMPTY_CLUSTER"),
+		os.Getenv("VSPHERE_NETWORK_LABEL"),
+		os.Getenv("VSPHERE_IPV4_ADDRESS"),
+		os.Getenv("VSPHERE_IPV4_PREFIX"),
+		os.Getenv("VSPHERE_IPV4_GATEWAY"),
+		os.Getenv("VSPHERE_DNS"),
+		os.Getenv("VSPHERE_DATASTORE"),
+		os.Getenv("VSPHERE_TEMPLATE"),
+		os.Getenv("VSPHERE_USE_LINKED_CLONE"),
 	)
 }

--- a/vsphere/resource_vsphere_virtual_machine_test.go
+++ b/vsphere/resource_vsphere_virtual_machine_test.go
@@ -803,10 +803,6 @@ func TestAccResourceVSphereVirtualMachine(t *testing.T) {
 						Config:      testAccResourceVSphereVirtualMachineConfigBasicEmptyCluster(),
 						ExpectError: regexp.MustCompile("compute resource .* is missing an Environment Browser\\. Check host, cluster, and vSphere license health of all associated resources and try again"),
 					},
-					{
-						Config: testAccResourceVSphereEmpty,
-						Check:  resource.ComposeTestCheckFunc(),
-					},
 				},
 			},
 		},

